docs/source/cli_help/advance.rst

```diff
@@ -14,7 +14,7 @@ advance
 
     git machete advance [-y|--yes]
 
-Fast forwards (as in ``git merge --ff-only``) the current branch ``C`` to match its downstream ``D``,
+Fast forwards (as in ``git merge --ff-only``) the current branch ``C`` to match its downstream ``D``, pushes ``C``
 and subsequently slides out ``D``. Both steps require manual confirmation unless ``-y/--yes`` is provided.
 
 The downstream ``C`` is selected according to the following criteria:
```

Comment:
My old mistake:
```suggestion
The downstream ``D`` is selected according to the following criteria:
```

Reply:
Also, `If --yes is specified...` is a part of the third point ` if C has more than one downstream...`, pls move